packages/dev-server-esbuild/src/EsbuildPlugin.ts

```diff
@@ -61,7 +62,15 @@ export class EsbuildPlugin implements Plugin {
     this.config = config;
     this.logger = logger;
     if (this.esbuildConfig.tsconfig) {
-      this.tsconfigRaw = await promisify(fs.readFile)(this.esbuildConfig.tsconfig, 'utf8');
+      try {
+        const { stderr, stdout } = await promisify(child_process.exec)(
```

Comment:
Does reading the file not work? We have to shell out to `tsc`?

Reply:
Oh, interesting! I'm wondering if we should really be using the `.tsconfig` setter instead of `.tsconfigRaw`. I'm not sure, but from reading the docs, it seems like that setting will work with extending configs?

If not, I think this is good!

Reply:
What do you mean with the `.tsconfig` setter? And are you referring to the dev-server-esbuild setting, or esbuild itself?

Reply:
@koddsson I just tested this again, and with the test case I added the test fails, without the change I've proposed here.